index.bs

```diff
@@ -9824,6 +9824,63 @@ Methods</h5>
 		</div>
 </dl>
 
+<h5 id="AudioWorkletProcessor-instantiation">
+The instantiation of AudioWorkletProcessor</h5>
+
+The construction of an {{AudioWorkletNode}} is followed by the construction
+of a corresponding {{AudioWorkletProcessor}} instance in the
+{{AudioWorkletGlobalScope}}.
+
+At the end of the {{AudioWorkletNode}} construction,
+<dfn>processor construction data</dfn> will be prepared for the cross-thread
+transfer. This data contains:
+
+<dl>
+	: <dfn dfn>nodeName</dfn>
+	::
+		A {{DOMString}} of the {{AudioWorkletProcessor}} name that is to
+		be looked up <a>node name to processor constructor map</a>.
+	: <dfn dfn>nodePort</dfn>
+	::
+		A serialized {{MessagePort}} that is paired with
+		the associated {{AudioWorkletNode}}'s {{AudioWorkletNode/port}}.
+	: <dfn dfn>nodeOptions</dfn>
+	::
+		A serialized {{AudioWorkletNodeOptions}} dictionary that is
+		given to the {{AudioWorkletNode}}
+		{{AudioWorkletNode()|constructor}}.
+</dl>
+
+Upon the arrival of the transferred data on the {{AudioWorkletGlobalScope}},
+the <a>rendering thread</a> will trigger the algorithm below:
+
+	<div id="invoking-processor-constructor" algorithm="invoking processor constructor">
+		Upon the arrival of <a>processor construction data</a>
+		<var>constructionData</var>, perform the following steps. If
+		any step throws an exception, abort the remaining steps and
+		<a>queue a task</a> to fire an
+		<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">
+		ErrorEvent</a> named <code>processorerror</code> at the
+		{{AudioWorkletNode}} on the <a>control thread</a> with the
+		relevant information about the error.
```

Comment:
This should probably be more clear about how the ErrorEvent is initialized....

Reply:
Do you think the "relevant information" bit is too loose?

Reply:
Well, what I meant is that it's really not clear what the message, filename, lineno, and colno attributes should _be_ in this case.   If the intent is that they are just `""`, `""`, `0`, and `0`, then we can just say "fire an event named `processorerror` using `ErrorEvent` at the AudioWorkletNode`, per <https://dom.spec.whatwg.org/#firing-events>.  If we want to initialize some of those attributes to specific values, we can do that similarly to the click event example given in that section.

Reply:
Thanks for the pointer. I'll follow your suggestion to clarify this.